chapters/3-measurement.tex

```diff
@@ -127,19 +127,24 @@ \subsection{Data linkage table}
 If one source of data will result in two different raw datasets,
 then list each dataset on its own row.
 For each dataset, list the unit of observation
-and the name of the project ID variable for that unit of observation.
+and the name of the \textbf{project ID}\sidenote{
+	\textbf{Project ID:} The main ID used in your project to identify
+	observations.
+	You should never have multiple project IDs for the same unit of observation.
+	The project ID must uniquely and fully identify all observations in the project.
+	See \url{https://dimewiki.worldbank.org/ID\_Variable\_Properties} for more details.}
+variable for that unit of observation.
 It is important to include both plain-language terminology
 as well as technical file and variable names here.
-For example, the \texttt{hh_baseline0_listmap.csv} dataset
+For example, the \texttt{hh\_baseline\_listmap.csv} dataset
```

Comment:
```suggestion
For example, the \texttt{hh\_baseline2020\_listmap.csv} dataset
```

The `0` was intentional to show extraneous information, is this better?

Reply:
ok, this is better. It look like a typo to me